pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -743,10 +743,19 @@ func TestDefaultPublicConfigMap(t *testing.T) {
 		{
 			name: "Test generating default public configmap with console URL",
 			want: &corev1.ConfigMap{
-				TypeMeta: metav1.TypeMeta{},
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "ConfigMap",
+					APIVersion: "v1",
+				},
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      api.OpenShiftConsolePublicConfigMapName,
 					Namespace: api.OpenShiftConfigManagedNamespace,
+					Annotations: map[string]string{
```

Comment:
We can drop these, since we are moving the manifest to be handled by console-operator